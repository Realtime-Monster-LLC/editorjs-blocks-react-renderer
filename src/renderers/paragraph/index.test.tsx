import React from 'react';
import { create } from 'react-test-renderer';
import Paragraph, { ParagraphBlockData } from '.';

describe('<Paragraph />', () => {
  describe('when receives a paragraph block', () => {
    const data: ParagraphBlockData = {
      text: 'Hey. Meet the new <b>Editor</b>. On this page you can see it in action â€” try to edit this text. Source code of the page contains the example of connection and configuration.',
    };

    it('renders a <p> tag', () => {
      expect(create(<Paragraph data={data} />).toJSON()).toMatchSnapshot();
    });
  });
});
