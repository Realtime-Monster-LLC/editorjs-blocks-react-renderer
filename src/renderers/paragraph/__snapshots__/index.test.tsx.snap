// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Paragraph /> when receives a paragraph block renders a <p> tag 1`] = `
<p>
  Hey. Meet the new 
  <b>
    Editor
  </b>
  . On this page you can see it in action â€” try to edit this text. Source code of the page contains the example of connection and configuration.
</p>
`;
